class-wc-gateway-komoju.php

```diff
@@ -135,6 +135,20 @@ public function process_payment($order_id)
         $payment_method = [sanitize_text_field($_POST['komoju-method'])];
         $return_url     = $this->get_mydefault_api_url();
 
+        // construct line items
+        $line_items = [];
+        foreach ($order->get_items() as $item) {
+          $image_parser = new DomDocument();
+          $image_parser->loadHTML($item->get_product()->get_image());
+          $img = $image_parser->getElementsByTagName('img')->item(0);
+
+          $line_items[] = [
+            'description' => $item->get_name(),
+            'quantity'    => $item->get_quantity(),
+            'image'       => $img->attributes->getNamedItem('src')->nodeValue,
+          ];
```

Comment:
KOMOJU supports per-item pricing info, but Woocommerce seems to have a pretty complex pricing system. There's item prices, tax, discounts, coupons, ... If we fail to emulate WC's pricing, then we'll end up creating payments with the wrong amount. So I'm thinking it's safest to exclude per-item pricing and just keep using the order total for session.amount.